public/js/controllers/ui_controller.js

```diff
@@ -114,6 +163,23 @@ export class UIController {
         UIController.statusBody.insertAdjacentHTML("beforeend", html);
     }
 
+    static addkeywordOptions(data) {
+        const statusMenu = document.getElementById("keyword");
+        if (statusMenu) {
+            while (statusMenu.firstChild) {
+                statusMenu.removeChild(statusMenu.firstChild);
+            }
+            let firstChildElement = `<option disabled selected>Filter By Active Projects</option>`
```

Comment:
clicking view all button should also reset the drop down back to Filter By Active Projects